docs/rules/multiline-html-element-content-newline.md

```diff
@@ -146,7 +146,9 @@ This rule enforces a line break before and after the contents of a multiline ele
 ## :couple: Related Rules
 
 - [vue/singleline-html-element-content-newline](./singleline-html-element-content-newline.md)
-- [no-multiple-empty-lines](https://eslint.org/docs/rules/no-multiple-empty-lines)
+- [no-multiple-empty-lines]
+
+[no-multiple-empty-lines]: https://eslint.org/docs/rules/no-multiple-empty-lines
```

Comment:
Why extract one of them but not both? I think extracting only really makes sense when the link is reused multiple times, so here, I'd keep it inlined.

Reply:
Since I introduced `@eslint/mardkown`, the following code has been flagged by `markdown/no-missing-label-refs`.

https://github.com/vuejs/eslint-plugin-vue/blob/a198b52b0ac26c8b8069fef1e295758ef7e5d932/docs/rules/multiline-html-element-content-newline.md?plain=1#L94

`[no-multiple-empty-lines]` was specified, but the link definition was not specified.
I changed it to use the link definition to resolve the warning.